### DISCUSSION TITLE
Android app, calendar - no longer possible to click on events from month view

### DISCUSSION THREAD

**Discussion by tsujigiri15** [tutao/tutanota#6576] [Ideas]
- [y ] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [ y] I've searched and did not find a similar issue.

**Bug in mobile app**

**Describe the bug**

**To Reproduce**
1. Go to 'Calendar, month view'
2. Click on 'an event'
3. Calendar opens to day view, rather than showing pop up of the event

Related - it is also no longer possible to add an event by highlighting a particular day in month view

**Expected behavior**
You used to be able to view event details from monthly view, rather than having to go into the day/agenda view. This is much more convenient when scheduling

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: pixel 4a
 - OS: CalyxOS, version 5.4 (latest update)
 - Version: Tutanota 3.122.4 (f droid build)

**Comment by charlag** [comment]
Hey, on mobile it was working like this for a very long time and it's intended this way. We will see if we can improve on that.
